.github/workflows/build-artifact.yaml

```diff
@@ -16,6 +16,10 @@ on:
         type: boolean
         default: false
         description: "Build with tracy enabled"
+      os:
+        required: false
+        type: string
+        default: "ubuntu-20.04-amd64"
   workflow_dispatch:
```

Comment:
Did we want to take away the ability to dispatch this particular workflow, as it requires docker artifact, and only allow `workflow_dispatch`ing the wrapper?

Reply:
technically this can still work if someone wants to build on main branch.

I think it would be better to remove the ability to call this though so will remove

Reply:
revised to allow for individually building docker-artifact.yaml
or just building tt-metal
or doing both